src/main/java/org/jabref/logic/openoffice/StyleLoader.java

```diff
@@ -16,14 +16,17 @@
 
 public class StyleLoader {
 
-    public static final String DEFAULT_AUTHORYEAR_STYLE_PATH = "/resource/openoffice/default_authoryear.jstyle";
-    public static final String DEFAULT_NUMERICAL_STYLE_PATH = "/resource/openoffice/default_numerical.jstyle";
+    public static final String DEFAULT_AUTHORYEAR_STYLE_PATH =
```

Comment:
It would be nice if you reformated changed code only.

Reply:
I agree.

The question of long lines comes back repeatedly.
I have 106 characters of horizontal space (about half width of the screen), and tend to break lines that are longer.
Idea would provide at most 183 (full screen, side bars closed), unless I find a smaller font.
The style rules seem to prefer long lines, and indeed, there are many above 100,
some even above 300. I was wondering how people manage to read those.

Are "fit-to-width" PRs something like the project would consider?
If yes, what would be a targeted limit on line length?
This may also raise: where to break.

Reply:
> I was wondering how people manage to read those.

I scroll 😛 
Are you asking about 'soft-wrap'? Screen-shot is from Intellij
<img width="424" alt="Skärmavbild 2021-04-23 kl  15 45 55" src="https://user-images.githubusercontent.com/2598631/115922688-32fee480-a44b-11eb-88f0-330bd9d163d1.png">

Reply:
It is possible. Breaking at white space instead of at the character at the edge of the window
helps somewhat, but still destroys the "visible indentation suggests code structure" effect.
My guess was they do no split their windows vertically, as Idea and probably other
IDEs put compilation results at the bottom under a horizontal split.

Reply:
> "visible indentation suggests code structure" effect.

I am not sure what you are referring to, first of all
1. are you using Intellij? 
2. does the below settings help?

<img width="1109" alt="Skärmavbild 2021-04-23 kl  16 56 50" src="https://user-images.githubusercontent.com/2598631/115928981-0059e980-a455-11eb-8873-f69d178c5c16.png">

> they do no split their windows vertically, as Idea

What is "Idea"? I split mine both horizontally and vertically o.O

Reply:
> 1.  are you using Intellij?

No. Not to edit code.

> 2.  does the below settings help?
> I am not sure what you are referring to, first of all

It seems you guessed correctly.
These settings improve the results by not destroying the outline.
This makes it more understandable why Intellij IDEA users seem to care less
about keeping a limit on their line lengths.

What it seems to miss when compared to hand-edited
breaks is a preference to break at higher points in the parse tree.
For example breaking between arguments of the outer call, not within.

Reply:
> For example breaking between arguments of the outer call, not within.

Indeed. I don't know of any options for this within Intellij.